admin_guide/managing_networking.adoc

```diff
@@ -1029,35 +1029,47 @@ xref:../install_config/configuring_sdn.adoc#install-config-configuring-sdn[confi
 to use the *ovs-networkpolicy* plug-in], network isolation is controlled
 entirely by
 link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/network/network-policy.md[`NetworkPolicy`
-objects]. In particular, by default, all projects are able to access pods in all
-other projects. To isolate a project, opt in to isolation by configuring the
-Namespace object, then create a `NetworkPolicy` object to indicate the allowed
-incoming connections.
+objects]. In particular, by default, all pods in a project are accessible from all sources.
+To isolate one or more pods in a project, you can create `NetworkPolicy` objects in that project
+to indicate the allowed incoming connections. Project administrators can create and delete
+`NetworkPolicy` objects within their own project.
+
+Pods that have no `NetworkPolicy` objects pointing to them are fully accessible, while
+Pods that have one or more `NetworkPolicy` objects pointing to them are isolated, and only
+accept connections that are accepted by at least one of their `NetworkPolicy` objects.
+This means you can make a project "deny by default" by adding a `NetworkPolicy` that
+matches all pods but accepts no traffic:
 
-. Configure the pod network in the master configuration file at *_/etc/origin/master/master-config.yaml_*:
-+
 [source,yaml]
 ----
-networkConfig:
- ...
-  networkPluginName: "redhat/openshift-ovs-networkpolicy" <1>
- ...
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
+metadata:
+  name: deny-by-default
+spec:
+  podSelector:
+  ingress: []
 ----
-<1> Set to *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 
-. Configure the pod network policy on each node configuration file at *_/etc/origin/node/node-config.yaml_*:
-+
+Alternatively, you could make pods accept connections from other pods in the same project,
+but reject all other connections from other projects:
```

Comment:
Should we mention that this would give the same behavior as with the multi-tenant plugin?

Reply:
It's not "the same"; it doesn't have the VNID 0 special case for instance. We could say it's "similar to" multitenant or something like that.